db/sqlanalyze.c

```diff
@@ -808,30 +830,32 @@ static int analyze_table_int(table_descriptor_t *td,
         sampled_table = 1;
         rc = sample_indicies(td, &clnt, tbl, td->scale, td->sb);
         if (rc) {
-            snprintf(sql, sizeof(sql), "Sampling table '%s'", td->table);
+            snprintf(zErrTab, sizeof(zErrTab), "Sampling table '%s'", td->table);
```

Comment:
Joe, can you please double-chec? I don't think zErrTab is populated in all reachable cases.